### ISSUE TITLE
Image does not have alternative text

### ISSUE THREAD

**Issue by zaymasood** [pulibrary/pulmap#993]
### Error in spec/features/accessibility_spec.rb
accessibility when visiting record show page complies with WCAG

       image-alt: Images must have alternate text (critical)
       https://dequeuniversity.com/rules/axe/4.3/image-alt?application=axeAPI
       The following 1 node violate this rule:
           
               Selector: .pul-logo > img
               HTML: <img src="/assets/pul-logo-new-744d8c02ecb992018c5d9ca281514c06d983db8d3baf7d1c9e18ab4df09f1e49.svg">
               Fix any of the following:
               - Element does not have an alt attribute
               - aria-label attribute does not exist or is empty
               - aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
               - Element has no title attribute
               - Element's default semantics were not overridden with role="none" or role="presentation"
